src/lib/reminderFields.ts

```diff
@@ -1,5 +1,4 @@
-import { EpicTargeting, EpicVariant } from '../types/EpicTypes';
-import { getUserCohorts } from '../tests/epics/epicSelection';
```

Comment:
this was causing the issue - it imports from a server-side file, which indirectly imports aws sdk